h/static/scripts/group-forms/utils/unsaved-changes.ts

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'preact/hooks';
+
+/** Count of components with unsaved changes. */
+let unsavedCount = 0;
```

Comment:
This is a non-reactive global variable so you can't use it to render different output depending on whether there are unsaved changes. We could address this if needed by making the value a signal.